### PR TITLE
flacon: init at 2.0.1

### PR DESCRIPTION
**Pull request by nico202**

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -6631,6 +6631,8 @@ in
 
   fcgi = callPackage ../development/libraries/fcgi { };
 
+  flacon = callPackage ../applications/audio/flacon { };
```
Comment (globin) [inline pkgs/top-level/all-packages.nix]: This should be added to the correct section, see other `../application/...` stuff

File: pkgs/applications/audio/flacon/default.nix [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+, shntool, flac, opusTools, vorbisTools, mp3gain, lame, wavpack, vorbisgain
+}:
+
+stdenv.mkDerivation rec {
+  name = "flacon-${version}";
+  version = "2.0.1";
+  src = fetchFromGitHub {
+    owner = "flacon";
+    repo = "flacon";
+    rev = "v2.0.1";
```
Comment (globin) [inline pkgs/applications/audio/flacon/default.nix]: You could reuse `version` here: `"v${version}"`

File: pkgs/applications/audio/flacon/default.nix [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    sha256 = "0hip411k3arb96rnd22ifs9shlv0xmy96hhx1jcwdk48kw8aa9rw";
+  };
+
+  buildInputs = [ cmake qt5.qtbase qt5.qttools libuchardet pkgconfig makeWrapper];
```
Comment (globin) [inline pkgs/applications/audio/flacon/default.nix]: OCD nitpick, there is a space after `[`, but none before `]` :scream:

File: pkgs/applications/audio/flacon/default.nix [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+  postInstall = ''
+    wrapProgram $out/bin/flacon \
+      --prefix PATH : "${shntool}/bin:${flac}/bin:${opusTools}/bin:\
```
Comment (globin) [inline pkgs/applications/audio/flacon/default.nix]: You can use `makeBinPath` here: https://github.com/NixOS/nixpkgs/blob/6f3e72eef6332ae988d213c76f77db65e8897e62/lib/strings.nix#L115-L123

Comment (nico202) [inline pkgs/applications/audio/flacon/default.nix]: Cool, didn't know of this!

**Comment by globin** [comment]
Thanks for the fast response! :beers:

**Pull request merged**